file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json

```
@@ -0,0 +1,20 @@
+{
+	"formatVersion": "1",
+	"header": {
+		"description": "AFF Example",
+		"originalLanguage": "en",
+		"abapLanguageVersion": "standard"
+	},
+	"implementation": {
+		"type": "CLAS",
+		"name": "ZCL_AFF_GSMP_TEST"
+	},
+	"execution": {
+		"mode": "system",
+		"position": "normal",
```

I think you changed `position` to `priority`, don't you?

```suggestion
		"priority": "normal",
```